src/Reader/readpar.f90

```diff
@@ -2164,69 +2161,50 @@ SUBROUTINE readpar_discretisation(EQN,MESH,DISC,SOURCE,IO)
-           END SELECT
-    CASE DEFAULT
-         logError(*) 'Wrong DGmethod. Must be 1 or 3.!'
-         call exit(134)
-    END SELECT
-       !
-    SELECT CASE(DISC%Galerkin%DGMethod)
-    CASE(1,3)
-           DISC%SpaceOrder = Order
-           if (DISC%SpaceOrder .ne. CONVERGENCE_ORDER) then
-                logWarning0(*) 'Ignoring min space order from parameter file, using', CONVERGENCE_ORDER
-           endif
-           DISC%SpaceOrder = CONVERGENCE_ORDER
-           DISC%Galerkin%nMinPoly = DISC%SpaceOrder - 1
-
-           if (DISC%SpaceOrder .ne. CONVERGENCE_ORDER) then
-                logWarning0(*) 'Ignoring space order from parameter file, using', CONVERGENCE_ORDER
-           endif
-           DISC%SpaceOrder = CONVERGENCE_ORDER
-           DISC%Galerkin%nPoly    = DISC%SpaceOrder - 1
-
-             ! The choice for p-adaptivity is not possible anymore
-             DISC%Galerkin%pAdaptivity = 0
-             logInfo(*) 'No p-Adaptivity used. '
-             logInfo(*) 'Basis functions degree:',DISC%Galerkin%nPoly
-
-           DISC%Galerkin%nPolyRec = DISC%Galerkin%nPoly
-           DISC%Galerkin%nPolyMat       = 0
-           DISC%Galerkin%nDegFrMat      = 1
-           DISC%Galerkin%nPolyMatOrig = Material
-           DISC%Galerkin%nPolyMatOrig = DISC%Galerkin%nPolyMatOrig - 1
-           logInfo(*) 'Material basis functions degree:',DISC%Galerkin%nPolyMatOrig
-           DISC%Galerkin%nPolyMap = nPolyMap
-           DISC%Galerkin%nPolyMat  = DISC%Galerkin%nPolyMatOrig + DISC%Galerkin%nPolyMap
-           DISC%Galerkin%nDegFrMat = (DISC%Galerkin%nPolyMat+1)*(DISC%Galerkin%nPolyMat+2)*(DISC%Galerkin%nPolyMat+3)/6
-           IF(DISC%Galerkin%nPolyMat.GT.DISC%Galerkin%nPoly) THEN
-             logError(*) 'nPolyMat larger than nPoly. '
-             call exit(134)
-           ENDIF
-
-           IF(MESH%GlobalElemType.EQ.6) THEN
-                  READ(IO%UNIT%FileIn,*) DISC%Galerkin%nPolyMatOrig
-                  READ(IO%UNIT%FileIn,*) DISC%Galerkin%nPolyMap
-                  DISC%Galerkin%nPolyMat  = DISC%Galerkin%nPolyMatOrig + DISC%Galerkin%nPolyMap
-                  DISC%Galerkin%nDegFrMat = (DISC%Galerkin%nPolyMat+1)*(DISC%Galerkin%nPolyMat+2)*(DISC%Galerkin%nPolyMat+3)/6
-                    IF(DISC%Galerkin%nPolyMat.GT.DISC%Galerkin%nPoly) THEN
-                         logError(*) 'nPolyMat larger than nPoly. '
-                         call exit(134)
-                    ENDIF
-           ENDIF
 
-    END SELECT
+    DISC%SpaceOrder = Order
+    if (DISC%SpaceOrder .ne. CONVERGENCE_ORDER) then
+         logWarning0(*) 'Ignoring min space order from parameter file, using', CONVERGENCE_ORDER
+    endif
+    DISC%SpaceOrder = CONVERGENCE_ORDER
+    DISC%Galerkin%nMinPoly = DISC%SpaceOrder - 1
+
+    if (DISC%SpaceOrder .ne. CONVERGENCE_ORDER) then
+         logWarning0(*) 'Ignoring space order from parameter file, using', CONVERGENCE_ORDER
+    endif
+    DISC%SpaceOrder = CONVERGENCE_ORDER
```

Comment:
This is redundant. 1. We set  `DISC%SpaceOrder` to `CONVERGENCE_ORDER` and then we check, if it's not equal and set it again.

Reply:
That's fine. I just marked the whole block, because marking two blocks and commeting on them at the same time does not really work...